### ISSUE TITLE
Any interest in me creating a "funnel" adventure?

### ISSUE THREAD

**Issue by RogueAutodidact** [MozillaFoundation/mpa-cryptomancer-challenge#20]
Hi all-
I really enjoy a particular type of unorthodox character creation called "the Funnel" (not my invention by the way). In the Funnel, each player gets a number of randomly generated "Level 0" characters. Level 0 characters aren't the badasses in the making that Level 1 characters are. Instead, they are ordinary folks caught in a bad (and very lethal) situation. 

A classic one is a merchant caravan being kidnapped by kobolds and locked a few levels down in a keep. Another is being caught in a town during a siege assault, and having to get out. Structure is usually involves a lot of running, half-clever plans and desperate fights (where the enemies are often superior & death is common) that all lead up to a climactic moment of actual heroism (or everyone dying). If they survive, the characters run off, but the seed of heroism is planted. They become heroes because of their experience, and the game proper starts with the aftermath/shared experiences from the Funnel informing the game. Or if all the poor *[people -.ed]* in the Funnel die (which _can_ happen) then the incident can be a reason for the main party to take notice of whatever took them out.

Any suggestions?

**Comment by chadsansing** [comment]
That's a cool framework, @RogueAutodidact - I'll keep an eye on the thread.

FYI - minor edit in alignment with our Code of Conduct.

**Comment by BakuDreamer** [comment]
Thinking about this, maybe you could have something like POV baton-pass while the action runs across two dozen or so characters, and then when it's over players can pick who they want to continue as.  " I'll be the guy who pushed the orc out the window " or " I'll be the elf who was running the bookstore "  and such.

**Comment by RogueAutodidact** [comment]
@BakuDreamer could you go into that? Would it be like the old Battlefield(?) mechanic where you would hop to a new soldier in an offensive if your current one died, rather than hitting a Game Over?

@chadsansing it will take me a little while to get through it- I am still trying to grok the rules system and difficulty levels well enough to make it quite difficult but not a certain party wipe. It can be a quite difficult balance- I've run versions of the Kobold in the Keep one where adding 2 more kobolds to the keep directly led to a party wipe, as the Level 0's couldn't maneuver enough and take the extra sling bullets. Also working out how to include mechanisms for basic shardnet use- probably low level stuff, like public Shardscape and maybe a stolen shard off a killed enemy to eavesdrop with. Any suggestions are quite welcome.

Also- @chadsansing sorry for the breach there. Haven't ever been on a formal collaborative/GitHub project before.

**Comment by BakuDreamer** [comment]
Yeah, that's what I was thinking, there'd be lots of NPCs all around, and if you get killed you can pick up as someone else and keep going.

**Comment by chadsansing** [comment]
Thanks for checking back in, @RogueAutodidact - as Mozilla and within this project we have some particular guidelines you can find in our Code of Conduct to keep everything as welcome and inclusive as possible.

Also, feel free to adjust the funnel a bit - maybe players trade characters as they get wiped? The big idea here is to empower players to make privacy and security choices in life that they learn from the game, so giving them a chance to iterate/get it right/succeed is huge for the project. I'm sure there's a way to combine those goals with a funnel set up that lets player see how commoners can make decisions that impact their lives and communities for the better.

**Comment by RogueAutodidact** [comment]
Maybe it could have less pandemonium. [] used to represent options.

In a small town (~500 people, say), people are going missing. No one knows what is causing the disappearances, but it seems to target families or groups of people who know each other. It will take several people at once, then the next group will have at least one person connected to the last group. If you mapped it out, a minor network cluster is taken out each time, then the next group is connected to members of that first cluster. It should be noted that when a group disappears, they aren't necessarily in the same geographical space- ie Alice might be on the High Street, her brother Bob at home, and their cousin Chuck chopping wood outside town. They do disappear at roughly the same time, and the interval of disappearances has been _quite_ regular- every 3 days. 5 groups of [Number of people at your table when you run this] people have disappeared so far. 

- There are a few clues left behind so far- like [echoes left on the shardscape by a victim describing a red flash/the only body found so far- a guard that wasn't on the list, but right next to where a targeted individual was staying, and he died in a distinctive way/ a door that was broken through]. 

- A group knows that they are chosen because [a distinct notch is carved in their door/a cleartext echo on the local shardscape naming them/other mark of GM's choice]. Townsfolk are realizing that being a close associate of a targeted person means that you might be chosen next, and are starting to shun the targeted person.

- When the group of commoners (The player party) runs out of time, the adversary will try to take them. It is much easier to do if the group has lax security protocols, bad communication among themselves, no basic crypto, etc. 

- If they coordinate, use their everyday talents to ferret out suspicious individuals, and take advantage of their closeness to use True Auth, they might survive that interval. But the next day, the signal marks them out once more. They have to solve the mystery or fall.

- If the group falls, the mark passes to the next group. 

My only issue is - how do these people get their own shardnet if they are commoners? Aren't they really, really expensive? 

Useful things this (potentially) intros:
- Social Network Analysis
- Splitting the Party using communications.
- Creeping Dread
- Mundane Tradecraft/Common Sense
- Shardscape Research 
- How the Shardscape is difficult
- Running down leads.
- True Auth (the group will already know each other well).

This seems to work somewhat better. It is still a bit awkward or forced to me, but this seems closer to design goals.

**Comment by chadsansing** [comment]
Maybe they formed a Shardnet co-op or someone worked some place that splits shards so they could pilfer a stray splinter?

**Comment by RogueAutodidact** [comment]
I think I might have a solution, and said solution would also easily adapt into demoing in school settings.

The setting is an elite private academy run by the local kingdom, teaching the children of the local aristocracy the basics and binding them together through shared scholastic suffering. Make each group (and each mini-shardnet) a study pod, created at the start of the student's term at the school and required throughout their term. Couple ways this can play out:
- The students are unaware of a hidden shard from each set, which the school (and now the attackers) monitor. 
    - Unsure if this makes sense given Trace and everything else can be done to figure out a network. 
    - School would use parallel construction to avoid disclosing the existence of the hidden shard.
- School golem has a shard from each set, allowing shardscape research for the students. 
    - Golem and/or cryptoadmin is compromised somehow.
    - Could allow for more network based finagling/tutorial if the whole school is arrayed in a network that is thoroughly compromised.

School is a symbol (and practical example of) the king's dominion over his nobility. Subjecting the nobility's children to danger and showing the state's inability to protect its vassals weakens the ties between nobility and the king. 

**Attacker End States:**
Dramatic Failure is: Being captured/killed and/or linked back to their patron.
Failure is: Being found out/driven off without being captured/killed or linked back to patron.
Success is: if the school is closed amid risk to students and bad blood is engendered between crown and nobility.
Exceptional success is: Open conflict between crown

@chadsansing - could Psychometry be used on a snatched student's shard? If typically only a student touched their own shard, wouldn't it be a handy investigation tool more generally? 

 Again, feels a tad artificial, but there are certain design constraints involved with having a funnel of semi-disposable characters w/o patrons and teaching players how to use equipment that is in-game expensive and slightly inflexible. By inflexible I mean that it does not appear easy to change the membership of a shardnet in any way beside physically transferring a shard.

**Comment by BakuDreamer** [comment]
Cute, sorta-military uniforms


​

**Comment by chadsansing** [comment]
Interesting take on student information management systems and snoopware on school machines IRL. @cryptomancer-actual - what do you make of the psychometry question above?

**Comment by cryptomancer-actual** [comment]
@RogueAutodidact 

I've read this thread. What I'm seeing start to form here is an adventure where "gifted" young people are sent off to some secret school for the gifted, where they learn how shards work, how shard spells, how eaves dropping works, how hand-to-hand combat works, how stealth works, and so on... until they (and the players) figure out the whole level 0 adventure is about how these gifted young people have been recruited by the Risk Eaters. Then it's time to escape. I'd help you write that adventure.

RE: Psychometry, doesn't work on shards. I put that in the spell's description.

@chadsansing I'm thinking a fair cut-off date for new content in the next Code & Dagger might be July 1st. Thoughts?

**Comment by chadsansing** [comment]
@cryptomancer-actual - yeah; let's pick a date - Maybe June 2nd, end of the Sprint, or were you thinking July? We can email & share here.

**Comment by BakuDreamer** [comment]
Some last minute entries ?!

New talents :

Appraisal ( spot fakes and forgeries as well as estimate values ),

Rich ( family or a trust or something )

Noble ( you have some kind of title )

Nondormenco ( sleepless, you don't sleep )

New cantrips :

Pattern Recognition : ( anomalies stand out like some kind of synesthesia )

Auction Chant : ( can talk so fast and for so long that a shard will '
buffer overflow ' maybe )

Bandersnatch : ( can produce truly random nonce words which can be added to
key-phrases as ' initialization vectors '  " Cazoova, Unelindo, Befurt,
Wardukers, Ketropial, ... "  The words are not on anyone's list or in use
anywhere else )

Suggestion :  " Roscoes " ( zap-guns )   Just some charges of the spell '
Kill Bolt ' in a shard spark-plug which is seated in a pistol-wand frame
that anyone can pick up and use.    ZAP !


​

**Comment by RogueAutodidact** [comment]
@cryptomancer-actual yeah, the level 0's may only have 6/6/4/4 for their attributes, maybe 1 talent? They are just coming up in the world, and maybe can earn 1 more talent to represent part of their training.

After the adventure (to bridge to a regular campaign), narrate/quick dice rolls about how they are on the run for a few weeks/months before they end with a patron in the campaign location. Depending on how they manage the Level 0, they start out with a bit of risk. 

There should be a moment where they hit an ethical wall that would force them to run away. Perhaps a "purge the village" kind of order? Risk Eaters regularly do terrible things, and I would be surprised if trainees were forced to some of them to weed out the squeamish/regrettably moral individuals. 

@BakuDreamer One thing for the Noble talent- iirc most characters will be incognito. Perhaps instead the character's noble background gives them a huge leg up in dealing with nobles. From balls to courtship to hard-nosed negotiations, they are simply better at working with nobles because _they are one_. Even if they have to fake a title, they can pull it off with the surety of acting like themselves, rather than putting on a show.

For rich- maybe they have difficult to access, illiquid, or hidden wealth. Maybe buried caches, discreetly owned properties (like a mine >50 miles away through a cutout), trade caravan investments etc. As a result, the character can't always rely on getting it right now, which allows the GM more control in granting resources. Given that significant amounts of coin would logically translate to advantages similar to certain strategic assets, GM's would likely value the ability to regulate it using in-game reasons.

@chadsansing One thing that could be amusing in terms of the school demos. I actually have a contact with a higher-up at GoGuardian, the leading K-12 snoopware and firewall provider. I could actually chat with him about how the whole thing works a little more deeply, and get some insight into the network design.

**Comment by fabiocosta0305** [comment]
If you have something like this already done, I would like to see and comment.
